Symfony/CS/Tests/Config/ConfigTest.php

```diff
@@ -51,7 +51,7 @@ public function testThatCustomFinderWorks()
         $config->finder($finder);
 
         $iterator = $config->getFinder()->getIterator();
-        $this->assertSame(1, count($iterator));
+        $this->assertCount(1, $iterator);
```

Comment:
thanks for tat one! 
I had no idea HHVM would fail because of this... wonder how longer it will take for this one to be fased out.. anyway.. reverted :)